src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -2305,14 +2305,6 @@ private void DeclareVariables(ImmutableArray<LocalSymbol> locals)
             }
         }
 
-        private void DeclareVariables(OneOrMany<LocalSymbol> locals)
```

Comment:
>DeclareVariables [](http://example.com/codeflow?start=21&length=16)

I am assuming the place that called this method silently calls the method above.

Reply:
Correct